src/openrct2/actions/TileModifyAction.hpp

```diff
@@ -95,11 +96,12 @@ DEFINE_GAME_ACTION(TileModifyAction, GAME_COMMAND_MODIFY_TILE, GameActionResult)
             return MakeResult(GA_ERROR::INVALID_PARAMETERS, STR_LAND_NOT_OWNED_BY_PARK);
         }
         auto res = MakeResult();
+
         switch (static_cast<TileModifyType>(_setting))
         {
             case TileModifyType::AnyRemove:
             {
-                const auto elementIndex = _value1;
+                uint32_t elementIndex = _value1;
```

Comment:
Like I said: please revert these changes that have nothing to do with your addition.